source/text/fontface.ts

```diff
@@ -202,12 +205,15 @@ export class FontFace {
 
     /**
      * The size of the font in pt. The font size is the measure from the tops of the tallest glyphs (ascenders) to
-     * the bottom of the lowest descenders in pt. It is derived via the sum of ascent and descent.
-     * @returns - The font size in pt (ascent + descent).
+     * the bottom of the lowest descenders in pt.
```

Comment:
is this pt, though? or rather an abstract texture space? Some font assets don't offer size in pt, but in px (e.g. openll asset generator)

Reply:
same for line below and for this whole file

Reply:
changed it for the whole file